src/Components/Products/index.js

```diff
@@ -1,10 +1,108 @@
 import React from 'react'
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
 import styles from './products.module.css';
+import Modal from '../Shared/Modal/index';
+
+const Products = (props) => {
+  const [products, setProducts] = useState([]);
+  const [showModal, setShowModal] = useState(false);
+  const [productId, setProductId] = useState(undefined);
+
+  useEffect(() => {
+    async function fetchProducts() {
```

Comment:
This function should be in a thunk and storing the response in redux not a local state

Reply:
To/do on other ticket